stock_move_location/wizard/stock_move_location.py

```diff
@@ -54,7 +40,9 @@ def _get_default_picking_type_id(self):
         column2="move_location_line_wiz_id",
     )
     picking_type_id = fields.Many2one(
-        comodel_name="stock.picking.type", default=_get_default_picking_type_id
+        compute="_compute_picking_type_id",
```

Comment:
@AungKoKoLin1997 The best is to use store=True too but maybe this is sufficent

Reply:
@rousseldenis I think we don't need store=True in here because it is in transient model and this attribute will do nothing.

Reply:
Transient are stored in DB

Reply:
@rousseldenis Yes. You are right. Thanks for the pointer. But I will not add this attribute in here because picking_type_id is not used in any domain or search.